### PR TITLE
Feature 1

### PR DESCRIPTION
**Pull request by TaiwoEnoch**
I work on my first project 'Hello Microverse' by testing all the tools needed in the program.
- Add index.html
- Add styles.css for styling
- Add linters also to avoid errors and to make my code dry.
- And all other tools also to make my code look professional.

### CODE REVIEW

File: test.md [hunk 1]
```diff
@@ -0,0 +1 @@
+This file should be ignored by git
```
Comment (mckent05) [inline test.md]: - According to the project requirements, this file should be ignored by git, hence shouldn't be visible in your repo. Please make sure that the `.gitignore` ignores this file 🙏. You might find this [article](https://www.freecodecamp.org/news/gitignore-file-how-to-ignore-files-and-folders-in-git/) helpful in knowing more about this 👍

File: README.md [hunk 2]
```diff
@@ -1,2 +1,50 @@
...
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
+
+## 📝 License
+
+This project is [MIT](./LICENSE) licensed.
+
+_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
```
Comment (mckent05) [inline README.md]: -Good job including a readme file that helps to make your repo more professional, but please use the new [microverse readme](https://github.com/microverseinc/readme-template), and ensure that it is personalized to fit what your project is all about. You are to change the microverse logo also to a personalized one if you have any 👍

**Review by mckent05 — CHANGES_REQUESTED** [review]
Hi @TaiwoEnoch, 

# STATUS: CHANGES REQUIRED ♻️ ♻️ 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
almost there

## Highlights
- All linter checks are passing ✔️ ✔️ 
- Your PR is professional 👍 👍 
- Good job using the right workflow for this milestone 👏 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 2 above)
Comment (TaiwoEnoch) [reply README.md]: Oh, thank you so much, I will work on it now, I am grateful.

File: test.md (hunk 1 above)
Comment (brainconnect93) [inline test.md]: - [ ] Great effort working on the README file as requested by the previous reviewer, 👏 👏  but I notice the `test.md` file issue raised by the reviewer has not been rectified yet. I can see you ignored it in the `.gitignore` file but it's still showing in your repo.
Kindly ignore it properly and be sure it's not showing again in your `repo` before you resubmit for review.
checkout this [article](https://www.freecodecamp.org/news/gitignore-file-how-to-ignore-files-and-folders-in-git/) for your peruse

**Review by brainconnect93 — CHANGES_REQUESTED** [review]
### Hi @TaiwoEnoch, 👋 

# STATUS: Changes Required :recycle:

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



### To Highlight :1st_place_medal: 
-  Has a proper Pull Request. ✔️  
- No Linter errors :heavy_check_mark: 
- Good use of Github flow ✔️ 

### Changes Required

- [x]  _Please check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @brainconnect93 ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: test.md (hunk 1 above)
Comment (TaiwoEnoch) [reply test.md]: Thank you, I will work on it now.

**Review by frank1738 — APPROVED** [review]
Hi @TaiwoEnoch , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

✅ Correct use of Github workflow
✅ No linter error

## Optional suggestions

N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by TaiwoEnoch** [comment]
Thanks so much for the review it is perfectly nice, just have to amend a slice mistake of not adding my styles.css link into my html file but have done it now immediately i realize it.
Thank so much for the review @frank1738

**Pull request merged**